deploy/helm/sumologic/README.md

```diff
@@ -275,6 +278,8 @@ The following table lists the configurable parameters of the Sumo Logic chart an
 | `telegraf-operator.data`                                                  | Telegraf sidecar configuration.                                                                                                                                                                                                                                                                                                                                                                                                               | `{"sumologic-prometheus": "[[outputs.prometheus_client]]\\n          ## Configuration details:\\n          ## https://github.com/influxdata/telegraf/tree/master/plugins/outputs/prometheus_client#configuration\\n          listen = ':9273'\\n          metric_version = 2\\n"}`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
+| `opentelemetry-operator.enabled`                                          | Flag to control deploying OpenTelemetry Operator Helm sub-chart.
+| `opentelemetry-operator.manager.env.WATCH_NAMESPACE`                      | Used to set value for WATCH_NAMESPACE environment variable which specifies Namespace to watch and create Instrumentation objects.
```

Comment:
Also this is three column table, and length should be formatted according to the convention, so

```
| a | long |
| b | c    |
```
instead of
```
| a | long |
| b | c |
```

Reply:
This is not fixed